src/components/Practice/Mechanics7.jsx

```
@@ -0,0 +1,703 @@
+  setOpenMessageDialog,
+  audio,
+  currentImg,
+}) => {
+  const [words, setWords] = useState(
+    type === "word" ? [] : ["Friend", "She is", "My"]
+  );
+  const [recordingStates, setRecordingStates] = useState({});
+  const [completeAudio, setCompleteAudio] = useState(null);
+
+  useEffect(() => {
+    if (words && words?.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)